Fix random bug on test_grab_with_tuple_percents()

The test `test_grab_with_tuple_percents()` randomly fails:
```python
    def test_grab_with_tuple_percents(sct):
        monitor = sct.monitors[1]
        left = monitor["left"] + monitor["width"] * 5 // 100  # 5% from the left
        top = monitor["top"] + monitor["height"] * 5 // 100  # 5% from the top
        right = left + 500  # 500px
        lower = top + 500  # 500px
        width = right - left
        height = lower - top
    
        # PIL like
        box = (left, top, right, lower)
        im = sct.grab(box)
        assert im.size == (width, height)
    
        # MSS like
        box2 = {"left": left, "top": top, "width": width, "height": height}
        im2 = sct.grab(box2)
        assert im.size == im2.size
        assert im.pos == im2.pos
>       assert im.rgb == im2.rgb
E       assert b"ffg8<@../.................." == b"ffg8<@../.................."
E         At index 84033 diff: b'\x00' != b'\xc0'
E         Full diff:
E           (
```

The cause is simple: the hour from the PIL screenshot is different from the one when the MSS screenshot is done.

A possible fix would be to repeat the test.

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -6,6 +6,7 @@ History:
       - removed usage of deprecated "license_file" option for "license_files"
       - fixed flake8 usage in pre-commit
       - MSS: the implementation is now thread-safe on all OSes (fixes #169)
+      - tests: fixed a random bug on test_grab_with_tuple_percents() (fixes #142)
       - :heart: contributors: @
 
 5.1.0   2020/04/30

--- a/setup.cfg
+++ b/setup.cfg
@@ -55,3 +55,6 @@ addopts =
     --failed-first
     -r fE
     -v
+    # Trait all tests as flaky by default
+    --force-flaky
+    --no-success-flaky-report

--- a/tox.ini
+++ b/tox.ini
@@ -10,6 +10,7 @@ skip_missing_interpreters = True
 passenv = DISPLAY
 alwayscopy = True
 deps =
+    flaky
     pytest
     # Must pin that version to support PyPy3
     pypy3: numpy==1.15.4

